CalibPPS/AlignmentGlobal/plugins/PPSAlignmentWorker.cc

```diff
@@ -91,7 +91,8 @@ class PPSAlignmentWorker : public DQMEDAnalyzer {
   edm::ESGetToken<PPSAlignmentConfiguration, PPSAlignmentConfigurationRcd> esTokenBookHistograms_;
   edm::ESGetToken<PPSAlignmentConfiguration, PPSAlignmentConfigurationRcd> esTokenAnalyze_;
 
-  edm::EDGetTokenT<CTPPSLocalTrackLiteCollection> tracksToken_;
+  std::vector<edm::InputTag> tracksTags_;
+  std::vector<edm::EDGetTokenT<CTPPSLocalTrackLiteCollection>> tracksTokens_;

```

Comment:
can these be `const`?

Reply:
Good idea, but only `tracksTags_` can be const. `tracksTokens_` cannot be initialised in the initializer list.